src/ybus_calculations.jl

```diff
@@ -7,16 +7,27 @@ struct Ybus{Ax, L <: NTuple{2, Dict}} <: PowerNetworkMatrix{ComplexF64}
     data::SparseArrays.SparseMatrixCSC{ComplexF64, Int}
     axes::Ax
     lookup::L
+    data_ft::SparseArrays.SparseMatrixCSC{ComplexF64, Int}
+    data_tf::SparseArrays.SparseMatrixCSC{ComplexF64, Int}
+    fb::Vector{Int64}
```

Comment:
To be as flexible as the existing struct these would type parameters:
```suggestion
    fb::Ax_fb
    tb::Ax_tb
```

though I think the existing struct is probably under-constrained, maybe it should be something like
```julia
struct Ybus{T} <: PowerNetworkMatrix{ComplexF64}
    data::SparseArrays.SparseMatrixCSC{ComplexF64, Int}
    axes::Tuple{Vector{T}, Vector{T}}
    lookup::Tuple{Dict{T, Int}, Dict{T, Int}}
    data_ft::SparseArrays.SparseMatrixCSC{ComplexF64, Int}
    data_tf::SparseArrays.SparseMatrixCSC{ComplexF64, Int}
    fb::Vector{T}
    tb::Vector{T}
end
```
but that would break backwards compatibility.

Reply:
Resolved.